docs/content/intro/community-roles.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Participating in the Athens Community"
+date: 2018-02-11T15:57:56-05:00
+---
+
+Absolutely everyone is welcome to join our community at any time! We
+are a friendly and inclusive group and we'd love to have you. We have three roles in the Athens community:
+
+- Community member
+- Contributor
+- Maintainer (we also call this core maintainer sometimes. They're the same thing)
+
+Read on to find out more!
+
+# Community Members
+
+Community members are folks who decide they want to get involved with our
+community. Absolutely anyone can do that whenever they want. If you want
+to get involved, that doesn't mean you have to continue being involved, but
+we hope our community is welcoming and the work is interesting enough to 
+convince you to say :)
+
+We'll provide all the support we can possibly provide to help you contribute
+in any way you'd like. If you're considering joining us, here are some ideas
+for how you can get involved:
```

Comment:
Maybe we can help with first contribution to open source? provide some mentorship. if something is up for it. i am

Reply:
In the kubernetes project, I wrote up guidelines for how to create an issue that is geared for new contributors.  When they look for issues labled `help wanted` and `good first issue` they have everything that they need to be successful, usually without directly contacting another maintainer for guidance (though it still makes it clear how to ask questions).

https://github.com/kubernetes/community/blob/master/contributors/devel/help-wanted.md

If people are interested, I highly encourage Athens to do this as well. It reduces the burden on the "ping people" and makes the project easier for people to onboard into because it doesn't assume that they are able/comfortable to lean on a mentor in private DMs, and moves a lot of institutional knowledge into writing, so that more people can learn how things are put together just by reading these issues.

Reply:
#562